index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms" action="https://index.html" method="post">
+    <p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <ul class="form_field">
+        <li class="form_name">
+            <input type="text" id="first_name" name="first_name" class="input" placeholder="Full Name" maxlength="30" required>
+        </li>
```

Comment:
- Following the project requirement, you should use [Formspree](https://formspree.io/html/) for submitting your contact inputs. Kindly make sure it is implemented correctly.
Hint: Your form should look like this:
```
<form class="contact_forms" action="https://formspree.io/f/{form_id}" method="post">
    <p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
    <ul class="form_field">
        <li class="form_name">
            <input type="text" id="first_name" name="first_name" class="input" placeholder="Full Name" maxlength="30" required>
        </li>
...
</form>
```
_Kindly go to [Formspree](https://formspree.io/html/) and generate your unique form_id._